mysql-test/extra/binlog_tests/enforce_gtid_consistency_tmp_violation.test

```diff
@@ -82,5 +84,6 @@ CREATE TEMPORARY TABLE IF NOT EXISTS t1 (a INT) ENGINE = MyISAM;
 SET AUTOCOMMIT = 0;
 --let $statement= DROP TEMPORARY TABLE t1
 --source extra/binlog_tests/enforce_gtid_consistency_statement.inc
+--let $sync_point= before_execute_sql_command
```

Comment:
Should this line be before `--source extra/binlog_tests/enforce_gtid_consistency_statement.inc`?

Reply:
before_execute_sql_command is the default upstream sync point here. The idea here was to restore it immediately after the part where this patch necessitates something else, both logically and for a smaller diff

Reply:
Oh, OK then.